rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/SpringSecurityToJaasFilter.java

```diff
@@ -68,7 +68,7 @@ protected Optional<Subject> obtainSubject() {
       return Optional.empty();
     }
     // TODO replace with Subject.current() when migrating to newer Version then 17
```

Comment:
```suggestion
    // TODO replace with Subject.current() when migrating to newer Version than 17
```

Reply:
I updated all the same comments in the code with your suggestion. :)
Thanks for finding my bad english spelling/writing issues :)